pkg/combustion/templates/09-time-setup.sh.tpl

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+{{ if .Timezone -}}
+ln -sf /usr/share/zoneinfo/{{ .Timezone }} /etc/localtime
+{{ end -}}
+
+{{ if or (gt (len .ChronyPools) 0) (gt (len .ChronyServers) 0) }}
+rm -f /etc/chrony.d/pool.conf
+{{ end -}}
+
+{{ range .ChronyPools -}}
+echo "pool {{ . }} iburst" >> /etc/chrony.d/eib-sources.conf
```

Comment:
Is having a new file all that's needed for this?

Reply:
Yep, if you look at `/etc/chrony.conf` it has an `includes` section so anything we throw in here, works:

```
localhost:~ # grep include /etc/chrony.conf
# Also include any directives found in configuration files in /etc/chrony.d
include /etc/chrony.d/*.conf

localhost:~ # cat /etc/chrony.d/eib-sources.conf
pool 2.suse.pool.ntp.org iburst
```

Chrony will then automatically pick up this pool:

```
localhost:~ # chronyc sources
MS Name/IP address         Stratum Poll Reach LastRx Last sample
===============================================================================
^+ time.cloudflare.com           3  10   377   341   +836us[ +993us] +/- 9231us
^* pool.ntp0.cam.ac.uk           2  10   377   328    -64us[  +93us] +/- 7829us
^- time.shf.uk.as44574.net       4  10   377   342   +449us[ +606us] +/-   28ms
^- ntp1.as200552.net             2   9   377   342   -973us[ -816us] +/-   46ms
```

Reply:
Thanks!